Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/MotionController/ControllerFinder.cs

```diff
@@ -1,125 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See LICENSE in the project root for license information.
-
-using Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities;
-using UnityEngine;
-
-namespace Microsoft.MixedReality.Toolkit.SDK.UX.MotionController
-{
-    /// <summary>
-    /// ControllerFinder is a base class providing simple event handling for getting/releasing controller transforms.
-    /// </summary>
-    public abstract class ControllerFinder : MotionControllerVisualizer
```

Comment:
What's the replacement?

(Also...why was this turned into a MotionControllerVisualizer?)

Reply:
This feature is superfluous in Alpha and was removed.  We will review going forward if it's actually required.